website_and_docs/content/documentation/webdriver/bidirectional/chrome_devtools/cdp_endpoint.en.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Chrome DevTools Protocol Endpoint"
+linkTitle: "CDP Endpoint"
+weight: 2
+description: >
+  Google provides a `/cdp/execute` endpoint that can be accessed directly. Each Selenium binding provides a method that
+  allows you to pass the CDP domain as a String, and the required parameters as a simple Map.
+---
+
+{{% pageinfo color="warning" %}}
+These methods will eventually be removed. It is recommended to use the [WebDriver-BiDi]({{< ref "../webdriver_bidi/" >}}) [WebDriver Bidi APIs]({{< ref "bidi_api.md" >}}) 
```

Comment:
Nit: Is there a conjunction missing between `[WebDriver-BiDi]({{< ref "../webdriver_bidi/" >}}) [WebDriver Bidi APIs]`? 
![Screenshot 2023-10-04 at 5 56 12 PM](https://github.com/SeleniumHQ/seleniumhq.github.io/assets/10705590/a86ea3ad-50c6-41c6-a82e-6f1132ef66f9)